data/json.go

```diff
@@ -158,6 +160,15 @@ func init() {
 }
 
 func newJsonConnector(filename string, l *common.Logger) (*jsonConnector, error) {
+
+	if _, db := os.Stat(filepath.Dir("db/")); os.IsNotExist(db) {
+		dbDir := os.Mkdir(filepath.Dir("db/"), 0777)
```

Comment:
`0777` might be overkill, please test if `0666` (or even better `0644`) still works